app/graphql/mutations.js

```diff
@@ -21,4 +21,13 @@ mutation Absences($classId: Int!, $studentId: Int!, $absences: Int!, $maxAbs: In
     message
   }
 }
+`
+
+export const registerUser = gql`
+mutation Register($registerBody: Register){
```

Comment:
```suggestion
mutation Register($registerBody: Register!){
```

Sin ese signo de admiración no va al ms y no sé por qué; se queda en el API